__tests__/analyzer/index.test.ts

```diff
@@ -1,5 +1,7 @@
 import type { OASDocument } from '../../src/rmoas.types';
 
+import { describe, beforeAll, it, expect } from 'vitest';
```

Comment:
Vitest offers a globals option[^1] for automatically adding these methods to every unit test but I kind of prefer the explicitness of **not** doing that.

[^1]: https://vitest.dev/config/#globals